winpr/libwinpr/sspi/test/TestSchannel.c

```diff
@@ -298,10 +297,9 @@ int schannel_recv(PSecurityFunctionTable table, HANDLE hPipe, PCtxtHandle phCont
 	ZeroMemory(&StreamSizes, sizeof(SecPkgContext_StreamSizes));
 	status = table->QueryContextAttributes(phContext, SECPKG_ATTR_STREAM_SIZES, &StreamSizes);
 	ioBufferLength = StreamSizes.cbHeader + StreamSizes.cbMaximumMessage + StreamSizes.cbTrailer;
-	ioBuffer = (BYTE*) malloc(ioBufferLength);
+	ioBuffer = (BYTE*) malloc(1, ioBufferLength);
```

Comment:
my fault, I think the compilation is OK because it's deadcode that is not compiled :)